src/datePicker/DatePicker.tsx

```diff
@@ -157,6 +157,7 @@ export class DatePicker extends PureComponent<DatePickerProps, DatePickerState>
     } = this.props
     const { date } = this.state
     const layoutClassName = `months-grid-${numberOfMonths}`
+    const initialMonth = orientation === DatePickerOrientation.VERTICAL ? new Date() : initialDate
```

Comment:
Could we just pass initialMonth as a nullable prop instead?
I think this logic shoud be handled in SPA, I'm not a fan of having a `new Date()`
Also how does that work with the `fromMonth` prop, can they conflict?

Reply:
Ok for me, I will directly handle this logic in SPA.

`fromMonth` refers to the first allowed month that users are allowed to navigate or interact. 
[fromMonth documentation](http://react-day-picker.js.org/api/DayPicker/#fromMonth)

`initialMonth` refers to the selected month that will be displayed in the calendar at first render.
[initialMonth documentation](http://react-day-picker.js.org/api/DayPicker/#initialMonth)